tools/wltests/platforms/akari_android-v4.9/build_images

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(dirname $(realpath -s $0))
+BASE_DIR="$SCRIPT_DIR/../.."
+source "${BASE_DIR}/helpers"
+source "${PLATFORM_PATH}/definitions"
+
+################################################################################
+# Utils
+################################################################################
+
+check_device_info() {
+    ERR_MSG="\
+Update the 'definitions' file if needed, but be aware that the values must \
+be in sync with the userspace currently flashed on the device or it won't boot\
+"
+    if [[ -z "$ANDROID_OS_VERSION" ]]; then
+	c_error "Could not read Android version from device"
+	c_error "$ERR_MSG"
+	exit $FATAL_ERROR
+    elif [[ -z "$ANDROID_OS_PATCH_LEVEL" ]]; then
+	c_error "Could not read Android patch level from device"
+	c_error "$ERR_MSG"
+	exit $FATAL_ERROR
+    fi
+}
+
+load_dtc_tools() {
+    # Find default tools from PATH
+    DTC_EXT_DEFAULT=$(which dtc)
+    DTC_OVERLAY_TEST_EXT_DEFAULT=$(which ufdt_apply_overlay)
+
+    # Override with user defined if required
+    export DTC_EXT=${DTC_EXT:-$DTC_EXT_DEFAULT}
+    export DTC_OVERLAY_TEST_EXT=${DTC_OVERLAY_TEST_EXT:-$DTC_OVERLAY_TEST_EXT_DEFAULT}
+
+    # Check for required tools being available
+    ERR_MSG="\
+It can be found at \
+https://android.googlesource.com/platform/prebuilts/misc/+/master/linux-x86/\
+"
+    if [[ ! -f "$DTC_EXT" ]]; then
+	c_error "Could not find [dtc] compilation tool"
+	c_error "Add it to your PATH or export it in DTC_EXT"
+	c_error "$ERR_MSG"
+	exit $FATAL_ERROR
+    fi
+    if [[ ! -f "$DTC_OVERLAY_TEST_EXT" ]]; then
+	c_error "Could not find [ufdt_apply_overlay] compilation tool".
+	c_error "Add it to your PATH or export it in DTC_OVERLAY_TEST_EXT"
+	c_error "$ERR_MSG"
+	exit $FATAL_ERROR
+    fi
+}
+
+################################################################################
+# Build all images required to test a new kernel
+################################################################################
+
+check_device_info
+load_dtc_tools
+
+echo
+c_info "Cleaning ${ARTIFACTS_PATH}..."
+rm -rf ${ARTIFACTS_PATH}
+
+export KCFLAGS=-mno-android
+export KBUILD_DIFFCONFIG=akari_diffconfig
+
+# The kernel is riddled with
+#     #include <something.h>
+# that should really be
+#     #include "something.h"
+# since those are local driver header files.
+# (see https://github.com/sonyxperiadev/kernel-copyleft/tree/51.1.A.2.xxx)
+#
+# As a tempfix, just use an outputdir
+OUTDIR="${KERNEL_SRC}"/out
```

Comment:
Should we set it to be under artifacts?

```bash
OUTDIR="${ARTIFACTS_PATH}/kernel_build"
```

Reply:
Oh yeah actually that could be a good way to have all of the artifacts there.